.changeset/tricky-trains-change.md

```diff
@@ -0,0 +1,8 @@
+---
+'example-app': minor
+'@backstage/core-components': minor
+'@backstage/plugin-catalog': minor
+---
+
+Displays errors in the entity page if the entity yaml is malformed
```

Comment:
We should put a bit more thought in here for people upgrading! Maybe something like this?

---

Added an `EntityProcessingErrorsPanel` component to show any errors that occurred when refreshing an entity from its source location. If upgrading, this should be added to `EntityPage` in your Backstage application:

```diff
// packages/app/src/components/catalog/EntityPage.tsx

const overviewContent = (
   ...
          <EntityOrphanWarning />
        </Grid>
      </EntitySwitch.Case>
+      <EntitySwitch.Case if={hasErrors}>
+        <Grid item xs={12}>
+          <EntityProcessErrors />
+        </Grid>
+      </EntitySwitch.Case>
    </EntitySwitch>
```

Additionally, `WarningPanel` now changes color based on the provided `severity`.

Reply:
That makes total sense, for some reason I thought this was autogenerated.